src/store/account/actions.js

```diff
@@ -4,29 +4,34 @@ export const login = async function (
   { commit, dispatch },
   { idx, account, returnUrl }
 ) {
-  const authenticator = this.$ual.authenticators[idx];
+  const authenticator = this.$ual().authenticators[idx];
   try {
     commit("setLoadingWallet", authenticator.getStyle().text);
     await authenticator.init();
+    console.log(account);
```

Comment:
logs

Reply:
Will remove all the console logs in the project.